Godeps/_workspace/src/bazil.org/fuse/syscallx/syscallx_std.go

```diff
@@ -3,24 +3,24 @@
 package syscallx
 
 // This file just contains wrappers for platforms that already have
-// the right stuff in stdlib.
+// the right stuff in golang.org/x/sys/unix.
 
 import (
-	"syscall"
+	"golang.org/x/sys/unix"
```

Comment:
@cryptix Not sure what you're talking about. That diff looks like commit 3c90b2396ff41bbe52a7b58b606d75cf47610a68 in bazil.org/fuse.